act/virtualNetwork.tf

```diff
@@ -17,89 +17,90 @@ resource "azurerm_resource_group" "resource_group" {
+  name                = "win-test"
+  location            = "East US"
   resource_group_name = "testrg"
   ip_configuration {
-    name = "internal"
-    subnet_id = azurerm_subnet.example.id
+    name                          = "internal"
+    subnet_id                     = azurerm_subnet.example.id
     private_ip_address_allocation = "Dynamic"
   }
 }
 
 resource azurerm_network_security_group "bad_sg" {
-  location = "East US"
-  name = "test-nsg"
+  location            = "East US"
+  name                = "test-nsg"
   resource_group_name = "testrg"
   security_rule {
-    access = "Allow"
-    direction = "Inbound"
-    name = "AllowSSH"
-    priority = 200
-    protocol = "TCP"
-    source_address_prefix = "*"
-    source_port_range = "*"
-    destination_port_range = "22-22"
+    access                     = "Allow"
+    direction                  = "Inbound"
+    name                       = "AllowSSH"
+    priority                   = 200
+    protocol                   = "TCP"
+    source_address_prefix      = "*"
+    source_port_range          = "*"
+    destination_port_range     = "22-22"
     destination_address_prefix = "*"
   }
   security_rule {
-    access = "Allow"
-    direction = "Inbound"
-    name = "AllowRDP"
-    priority = 300
-    protocol = "TCP"
-    source_address_prefix = "*"
-    source_port_range = "*"
-    destination_port_range = "3389-3389"
+    access                     = "Allow"
+    direction                  = "Inbound"
+    name                       = "AllowRDP"
+    priority                   = 300
+    protocol                   = "TCP"
+    source_address_prefix      = "*"
+    source_port_range          = "*"
+    destination_port_range     = "3389-3389"
     destination_address_prefix = "*"
   }
 }
 
 resource "azurerm_storage_account" "my_storage" {
-  name                     = "functionsapptestsa"
-  resource_group_name      = azurerm_resource_group.resource_group.name
-  location                 = azurerm_resource_group.resource_group.location
-  account_tier             = "Standard"
-  account_replication_type = "LRS"
+  name                      = "functionsapptestsa"
+  resource_group_name       = azurerm_resource_group.resource_group.name
+  location                  = azurerm_resource_group.resource_group.location
+  account_tier              = "Standard"
+  account_replication_type  = "LRS"
+  enable_https_traffic_only = true
 }
-  
+
 resource "azurerm_app_service_plan" "app_service_plan" {
   name                = "azure-functions-test-service-plan"
   location            = azurerm_resource_group.resource_group.location
   resource_group_name = azurerm_resource_group.resource_group.name
   kind                = "FunctionApp"
-  
+
   sku {
     tier = "Dynamic"
     size = "Y1"
   }
 }
-  
+
 resource "azurerm_function_app" "function_app" {
   name                       = "test-azure-functions"
   location                   = azurerm_resource_group.resource_group.location
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure Storage Account is using the latest version of TLS encryption'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

 <!-- 3ec36ae0-3fe5-49f4-92a7-ee656c9c7fad -->